app/views/events/add_services.html.haml

```diff
@@ -1,3 +1,23 @@
+<<<<<<< HEAD
```

Comment:
?

Reply:
whoops merge conflict apparently didnt break the build

Reply:
Wouldn't break until you tried to render the page.

Running a test for this view would also have pointed out this error.